scripts/crawler.js

```diff
@@ -18,8 +18,25 @@ const V8_NATIVE_CALL_REPLACEMENT_PREFIX = '__v8_native_remainder'
 const V8_NATIVE_CALL_REPLACEMENT_REGEX = /__v8_native_remainder(\w+\(\S*?|\s*\))/gm
 const V8_NATIVE_CALL_FLAGS_COMMENT_REGEX = /\/\/\s*Flags:.*(--allow-natives-syntax)+/gm
 
-const GLOBAL_METHODS =
-  ";(function(globals){globals._ddiast = globals._ddiast || {plusOperator(res) {return res;}};}((1,eval)('this')));"
+const CSI_METHODS = [
+  { src: 'plusOperator' },
+  { src: 'substring' },
+  { src: 'trim' },
+  { src: 'trimStart' },
```

Comment:
We should add trimLeft and trimRight since they are alias for trimStart and trimEnd respectively.